cmdutils/progress.go

```diff
@@ -4,29 +4,41 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/jedib0t/go-pretty/text"
 	"github.com/vbauerster/mpb/v5"
 	"github.com/vbauerster/mpb/v5/decor"
 )
 
 const (
-	progressCounterFormat         = "%d / %d ["
-	spinnerCounterFormat          = "%d ["
+	progressCounterFormat         = "%d / %d "
+	spinnerCounterFormat          = "%d "
+	spinnerNoCounterFormat        = ""
 	progressSuffix                = "]"
+	progressPrefix                = "["
 	progressBarWidth              = 60
 	progressBarNameColumnWidth    = 40
 	progressBarCounterColumnWidth = 20
 	progressBarStyle              = " =>- <"
 	progressRefreshRate           = 50 * time.Millisecond
 )
 
+const (
+	Bar = iota
+	Spinner
+	SpinnerNoCounter
+)
+
 type ProgressReporter interface {
 	Progress() []*Progress
 }
 
 type Progress struct {
-	label   string
-	current *int64
-	total   *int64
+	label        string
+	current      *int64
+	total        *int64
```

Comment:
Think that it will be better to have these as `int64` and when we call the atomic increment get the address of the variable - so we will not be required to handle nil.